### PR TITLE
docs(readme): add requirements section INTER-276

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

**Comment by ilfa** [comment]
@Orkuncakilkaya can you please test compatibility with the latest Angular version?

File: README.md [hunk 1]
```diff
@@ -37,13 +37,23 @@ Fingerprint Pro Angular SDK is an easy way to integrate  **[Fingerprint Pro](htt
...
 - [Caching strategy](#caching-strategy)
 - [Documentation](#documentation)
 - [Support and feedback](#support-and-feedback)
 - [License](#license)
 
+## Requirements
+
+The following dependencies are required:
+
+- TypeScript 4
```
Comment (JuroUhlar) [inline README.md:52]: I presume it's a minimal TypeScript version, right? Or is it not usable with TypeScript 5.x?

```suggestion
- TypeScript 4+
```

Comment (Orkuncakilkaya) [reply README.md:52]: Going to check with Typescript 5 if it's working.

**Comment by Orkuncakilkaya** [comment]
This integration works for Angular version between 13 and 17 (latest)
Angular 17 only works with Typescript >=4.9.3 <5.3.0. So it doesn't supports Typescript 5.3.0 or 5.3.3 versions of Typescript.
I updated our libraries requirements as:
```
- TypeScript >=4.6 and <= 5.2
- Angular 13+
```

CC: @JuroUhlar @ilfa

**Pull request closed without merging**